airflow/models/mappedoperator.py

```diff
@@ -812,6 +812,8 @@ def unmap(self, resolve: None | Mapping[str, Any] | tuple[Context, Session]) ->
             op.is_setup = is_setup
             op.is_teardown = is_teardown
             op.on_failure_fail_dagrun = on_failure_fail_dagrun
+            op.downstream_task_ids = self.downstream_task_ids
+            op.upstream_task_ids = self.upstream_task_ids
```

Comment:
@uranusjr following up https://github.com/apache/airflow/issues/43883#issuecomment-2527022673 - it seems that these attributes were missing here, so that's why PythonOperator printed misleading logs